ui-components/src/locales/general.json

```diff
@@ -1382,6 +1382,10 @@
       "studio": "Studio",
       "SRO": "SRO"
     },
+    "vacantUnit": "vacant unit",
+    "vacantUnits": "vacant units",
+    "unitSummaryGroupMessage": "For each type of unit, you cannot make more than the income limit shown for the",
```

Comment:
Ideally we don't concat strings like this for translations where these concats won't make sense

Reply:
I agree. I'm going to leave it like this for now and think of a better way to handle links within translated text.